bookwyrm/views/admin/reports.py

```diff
@@ -81,41 +82,42 @@ def get(self, request, report_id):
     def post(self, request, report_id):
         """comment on a report"""
         report = get_object_or_404(models.Report, id=report_id)
-        models.ReportComment.objects.create(
-            user=request.user,
-            report=report,
-            note=request.POST.get("note"),
-        )
+        note = request.POST.get("note")
+        report.comment(request.user, note)
         return redirect("settings-report", report.id)
 
 
 @login_required
 @permission_required("bookwyrm.moderate_user")
-def suspend_user(request, user_id):
+def suspend_user(request, user_id, report_id=None):
```

Comment:
It seems strange to me that the suspended user is not notified - they should probably get an email telling them their account is suspended due to a moderation decision?

Reply:
I'm not sure! Almost all of the users I've suspended so far have been spam accounts, and I'm not sure if it's counterproductive to notify them? But I see how it _does_ make sense in a case of an actual person who has been behaving poorly. I think this is out of scope of this PR, however. I'll open an issue for it.